python/src/RatioOfUniforms_doc.i.in

```diff
@@ -0,0 +1,487 @@
+%feature("docstring") OT::RatioOfUniforms
+"Ratio of Uniforms simulation algorithm.
+
+
+.. warning::
+    This class is experimental and likely to be modified in future releases.
+    To use it, import the ``openturns.experimental`` submodule.
+
+Available constructors:
+    RatioOfUniforms(*dist*)
+
+    RatioOfUniforms(*logUnscaledF, rangeF, isScaled*)
+    
+Parameters
+----------
+dist : :class:`~openturns.Distribution`
+    Distribution to sample.
+    
+    With that constructor, the range is the distribution range and *isScaled* = True.
+
+logUnscaledPDF : :class:`~openturns.Function`
+   Function :math:`\log F` such that :math:`f = F\, 1_{I}`  is proportional to the PDF :math:`p` we want to sample.
+
+rangeF : :class:`~openturns.Interval`
+    Interval :math:`I \subset \Rset^\inputDim` outside which :math:`p=0`.
+
+isScaled : bool, optional
+    Flag telling if the input *logUnscaledPDF* is such that :math:`f = F\, 1_{I}` is an actual PDF of a distribution.
```

Comment:
idem bold

Reply:
yes. I checked all the instances of 1_...